util/xtpm_read_nvram.c

```diff
@@ -0,0 +1,174 @@
+/******************************************************************************
```

Comment:
The utility isn't really `xtpm_read_nvram`, since it's only capable of reading 6 specific object. The `xtpm_read_nvram` can read any `HANDLE` (that has the appropriate permissions).

Should this be renamed to `xtpm_read_object`?   (I don't like the word `object`, but haven't thought of something more descriptive.)

Reply:
I'll name the `enum`-able function (from your above comment) also called `xtpm_read_object`, as well as this utility executable.

I can't think of anything better than "object", and it doesn't bother me too much...